src/main/jbake/content/blog/2016/07/14/hawkular-miq.adoc

```diff
@@ -0,0 +1,109 @@
+= Getting started with ManageIQ and Hawkular
+Heiko W. Rupp
+2016-07-14
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, manageiq
+
+This post describes the first steps to get started with ManageIQ master and Hawkular-services.
+
+
+== Setup of ManageIQ
+
+The functionality to support the Hawkular-Middleware provider is not in the Darga-Release.
+You will need to build ManageIQ from their master branch.
+There is a good http://manageiq.org/documentation/development/developer_setup/[guide on developer setup].
+
+Once you are done, you can start ManageIQ like this:
+
+.Start ManageIQ
+[source,shell]
+--
+$ bundle exec rake evm:start
+--
+
+The server will listen on `http://localhost:3000`.
+
+== Setup of Hawkular-services
+
+Run Hawkular-services with some WildFly servers connected. While it is possible to only run Hawkular-services with the built-in agent enabled, you will get more interesting results if you point other instrumente WildFly servers at Hawkular.
+
+=== Docker
+If you are a Docker user, you can http://pilhuhn.blogspot.de/2016/06/using-hawkular-services-via-docker.html[follow this route]. This uses Docker-compose to stand up Cassandra 3.7, Hawkular-services and a connected WildFly10 server.
```

Comment:
> imo worth to mention somehow metrics? because otherwise there will be 
> no metrics nor reports in miq..

Not sure I get that. Hawkular-metrics is part of hawkular-services.
And making sure an agent is connected (the docker-compose even starts 2 
- internal and external)
should create metric data.